code/modules/uplink/uplink_items.dm

```diff
@@ -504,6 +504,16 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	manufacturer = /datum/corporation/traitor/waffleco
 	exclude_modes = list(/datum/game_mode/infiltration) // yogs: infiltration
 
+/datum/uplink_item/dangerous/vxtvulhammer
+	name = "Vxtvul Hammer"
+	desc = "The Vxtvul Hammer is a sledgehammer once utilized by the ancient Vxtrin species. \
+			This weapon must be wielded in two hands to be used effectively, but possesses high armor penetration. \
+			In addition, the user can charge the hammer to enable a thunderous blow that will decimate construction in a single hit, \
+			do sizeable damage to mechs, or shatter people off of their feet. The battery is charged by the user's concentration."
+	item = /obj/item/twohanded/vxtvulhammer
+	cost = 13
+	include_modes = list(/datum/game_mode/nuclear) //Only traitor preterni can buy the implant version
```

Comment:
tested?

Reply:
Which part do you mean tested? The hammer's functionality? In a live round? It appearing in uplink?